### ISSUE TITLE
RazorPagesMovie - Several issues solved - Biggest is with Codegenerator for CRUD

### ISSUE THREAD

**Issue by beamand** [dotnet/AspNetCore.Docs#27546]
Using Version 7.0.0 or a mixture of versions 6.0.* of Microsoft.EntityFrameworkCore.Design caused one of these errors:

![Screenshot 2022-11-08 122236](https://user-images.githubusercontent.com/101301345/200666599-9d5c69f3-4df5-4442-a2a9-d2cb17d01d02.jpg)

![Screenshot 2022-11-08 122029](https://user-images.githubusercontent.com/101301345/200668040-7776ab0b-2fe3-44aa-947f-bb6bf6587f83.jpg)

This issues was solved by going back to version 6.0.10 of all Microsoft.EntityFrameworkCore.* 

![image](https://user-images.githubusercontent.com/101301345/200667308-c37f7851-2ad0-4419-b091-ff8dc2e63334.png)

In the following image you will notice that when using Version 7.0.0 and you try generating the CRUD, that the ".Models" is missing from then end of the Model class:

![Screenshot 2022-11-08 133649](https://user-images.githubusercontent.com/101301345/200673330-da99d27f-074d-4422-806b-e0c78e752562.jpg)

Versus

![Screenshot 2022-11-08 122130](https://user-images.githubusercontent.com/101301345/200673468-24ae12ba-576e-40a0-b09c-db127e2dfb2b.jpg)

So I conclude that is causing the path issue to the .Models directory and Movie.cs code going to V6 to V7.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6719f08e-3bd7-dc1a-71df-f2ef9fbca9d8
* Version Independent ID: 7096fdb3-612e-9e00-bd0b-8ea4886a09ce
* Content: [Part 2, add a model](https://learn.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-7.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/model.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/model.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Exit and restart VS and it works. Transient error.

**Comment by beamand** [comment]
I finally got it to work thanks and finished the tutorial but there is one
item left hanging.

Where in the code would be the Best Place to Trim() the "Title" and send
back an error if the length is not greater than 3?
I tried in the Movie class but it does not like me changing the get; set;
because of some auto... setting error.
The other place I have not tried is in the Create & Edit after the
(!ModelState.IsValid).

Thanks,
Danny
